src/libs/sign.js

```diff
@@ -177,8 +177,9 @@ export const signxcarchive = async (archiveFilePath, workspace = '/tmp/') => {
     const outputDir = 'signed';
     const apath = await extractArchiveContents(archiveFilePath, workspace);
     const findResult = await exec(`find ${apath} -iname '*.xcarchive'`);
-    if (findResult.stderr !== '') {
-      throw new Error('Unable to find xcarchive(s) in package');
+    const plistPath = await exec(`find ${apath} -iname 'options.plist'`);
+    if (findResult.stderr || plistPath.stderr !== '') {
```

Comment:
Do you want `&&` here? If `stderr` exists _and_ its not and empty string?

Reply:
Oh I make a mistake here, should be just (findResult.stderr || plistPath.stderr) . As we want both package and plist to exist.